src/cbmc_starter_kit/template-for-repository/proofs/run-cbmc-proofs.py

```diff
@@ -143,6 +145,13 @@ def get_args():
             "flags": ["--debug"],
             "action": "store_true",
             "help": "debug output",
+    }, {
+            "flags": ["--tabular-summary-print-fmt"],
+            "type": str,
+            "choices": tabulate.tabulate_formats,
+            "metavar": "FORMAT",
```

Comment:
This gets printed out twice so should be as short as possible, e.g. `F` or `FMT`